zerver/tests/test_push_notifications.py

```diff
@@ -1682,7 +1682,8 @@ def test_backend_only_fixtures(self) -> None:
         fixtures = [
             {
                 'name': 'realm_emoji',
-                'rendered_content': '<p>Testing <img alt=":green_tick:" class="emoji" src="/user_avatars/1/emoji/green_tick.png" title="green tick"> realm emoji.</p>',
+                'rendered_content': '<p>Testing <img alt=":green_tick:" class="emoji" src="/user_avatars/%s/emoji/green_tick.png" title="green tick"> realm emoji.</p>' % (
+                    get_realm("zulip").id,),
```

Comment:
Let'd add a block at the top where we get values like this; it'll look cleaner and we ideally need similar ones for the `data-user-id` and `data-stream-id` in the next couple tests.

Reply:
Added the block.